trunk/SUAVE/Methods/Aerodynamics/Fidelity_Zero/Drag/compressibility_drag_wing.py

```diff
@@ -64,26 +74,31 @@ def compressibility_drag_wing(state,settings,geometry):
         cl_w = wing_lifts
     else:
         cl_w = 0
-        
-    cos_sweep = np.cos(sweep_w)
 
     # get effective Cl and sweep
-    tc = t_c_w /(cos_sweep)
-    cl = cl_w / (cos_sweep*cos_sweep)
+    tc = t_c_w /(np.cos(sweep_w))
+    cl = cl_w / (np.cos(sweep_w))**2
+    
+
+
+
+
+    #-----------Existing SUAVE-----------------------------------
+
 
     # compressibility drag based on regressed fits from AA241
     mcc_cos_ws = 0.922321524499352       \
                - 1.153885166170620*tc    \
                - 0.304541067183461*cl    \
-               + 0.332881324404729*tc*tc \
+               + 0.332881324404729*tc**2 \
                + 0.467317361111105*tc*cl \
-               + 0.087490431201549*cl*cl
+               + 0.087490431201549*cl**2
         
     # crest-critical mach number, corrected for wing sweep
-    mcc = mcc_cos_ws / cos_sweep
+    mcc = mcc_cos_ws / np.cos(sweep_w)
```

Comment:
Everything in this file and induced came from Anil's files. He probably was using an older version to start with. We can update these manually to match.